keywords/TestServeriOS.py

```diff
@@ -0,0 +1,323 @@
+            self.package_name = "CBLTestServer-iOS-enterprise-{}.zip".format(version_build)
+            self.app = "CBLTestServer-iOS-EE"
+
+    def download(self, version_build=None):
+        """
+        1. Check to see if package is downloaded already. If so, return
+        2. Download the LiteServ package from latest builds to 'deps/binaries'
+        3. Unzip the packages and make the binary executable
+        """
+        if version_build is not None:
+            self.version_build = version_build
+        version, build = version_and_build(self.version_build)
+        app_name = "{}-{}.app".format(self.app, version_build)
+
+        expected_binary_path = "{}/{}/{}".format(BINARY_DIR, self.app_dir, app_name)
+        if os.path.isfile(expected_binary_path):
+            log_info("Package is already downloaded. Skipping.")
+            return
+
+        # Package not downloaded, proceed to download from latest builds
+        downloaded_package_zip_name = "{}/{}".format(BINARY_DIR, self.package_name)
+        url = "{}/couchbase-lite-ios/{}/{}/{}".format(LATEST_BUILDS, version, build, self.package_name)
+
+        log_info("Downloading {} -> {}/{}".format(url, BINARY_DIR, self.package_name))
+        resp = requests.get(url)
+        resp.raise_for_status()
+        with open("{}/{}".format(BINARY_DIR, self.package_name), "wb") as f:
+            f.write(resp.content)
+
+        extracted_directory_name = downloaded_package_zip_name.replace(".zip", "")
+        with ZipFile("{}".format(downloaded_package_zip_name)) as zip_f:
+            zip_f.extractall("{}".format(extracted_directory_name))
+
+        # Remove .zip
+        os.remove("{}".format(downloaded_package_zip_name))
+
+    def install_device(self):
+        """Installs / launches CBLTestServer on iOS device
+        Warning: Only works with a single device at the moment
+        """
+        if self.debug_mode:
+            self.app_name = "{}-{}-Device-debug.app".format(self.app, self.version_build)
+        else:
+            self.app_name = "{}-{}-Device.app".format(self.app, self.version_build)
+        # package_name = "CBLTestServer-iOS-Device.app"
+        # app_dir = "CBLTestServer-iOS"
+
+        self.app_path = "{}/{}/{}".format(BINARY_DIR, self.app_dir, self.app_name)
+        log_info("Installing: {}".format(self.app_path))
+
+        # install app / launch app to connected device
+        output = subprocess.check_output([
+            "ios-deploy", "--justlaunch", "--bundle", self.app_path
+        ])
+        log_info(output)
+
+        bundle_id = "com.couchbase.CBLTestServer-iOS"
+        output = subprocess.check_output(["ios-deploy", "--list_bundle_id"])
+        log_info(output)
+
+        if bundle_id not in output:
+            raise LiteServError("Could not install CBLTestServer-iOS")
+
+        self.stop()
+
+    def install(self):
+        """Installs / launches CBLTestServer on iOS simulator
+        """
+        # self.device = "iPhone-7-Plus"
```

Comment:
remove the commented code, if not needed.

Reply:
done